WordPress/src/main/java/org/wordpress/android/ui/main/WPMainTabLayout.java

```diff
@@ -64,8 +63,8 @@ private void addTab(@DrawableRes int iconId, @StringRes int contentDescriptionId
      * unread notifications
      */
     void checkNoteBadge() {
-        showNoteBadge(SimperiumUtils.hasUnreadNotes());
-    }
+        showNoteBadge(false);
```

Comment:
Ok, I've investigated it a bit more, and seems that the badge represents the notifications unseen status. Not the unread one, that is per note. 
Working on providing a patch for it.